### ISSUE TITLE
Accessbility - JS - Tabs: Keyboard

### ISSUE THREAD

**Issue by Ibabalola** [Capgemini/dcx-react-library#658]
For accessbility, add functionality to allow the user to tab between tabs using the key only (when JavaScript is enabled), currently the user is unable to view inactive tabs without the use of a keyboard.

'ArrowLeft', 'ArrowUp', 'Left' and 'Up': should take user to the previous tab (if on tab 2 or more)

'ArrowRight', 'ArrowDown', 'Right' and 'Down': should take user to the next tab (if not on the last tab)

As an example please check keyboard behaviour on:
[GOV.UK - tabs](https://design-system.service.gov.uk/components/tabs/default/)

**Comment by alexwbbr** [comment]
Closed as part of https://github.com/Capgemini/dcx-react-library/pull/659
